source/opt/passes.h

```diff
@@ -58,6 +58,14 @@ class StripDebugInfoPass : public Pass {
   bool Process(ir::Module* module) override;
 };
 
+class FreezeSpecConstantValuePass : public Pass {
+ public:
+  const char* name() const override {
+    return "Freeze spec constants to their default value";
```

Comment:
Changed StripDebugInfoPass::name() to return "strip-debug"
Changed FreezeSpecConstantValuePass::name() to return "freeze-spec-const"
Both match with their command line arguments